src/security/cli_security.c

```diff
@@ -1,24 +1,7 @@
 /*
  * (C) Copyright 2018-2020 Intel Corporation.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- *
- * GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
- * The Government's rights to use, modify, reproduce, release, perform, display,
- * or disclose this software are subject to the terms of the Apache License as
- * provided in Contract No. 8F-30005.
- * Any reproduction of computer software, computer software documentation, or
- * portions thereof marked with this legend must also reproduce the markings.
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
```

Comment:
Huh, we don't need the full text of the license in these headers? Also, do we not need any USG info like the contract number?

Reply:
We likely need to restore the government contract section, working with Intel legal team to get their input to the statement. Will update this PR when we get there.

Reply:
The solution after discussion with the legal team is that we don't need to have the government license section in DAOS file header since those contracts don't require license info.